### DISCUSSION TITLE
Panda 0.37 breaks recipes that were working on 0.31

### DISCUSSION THREAD

**Discussion by jderboven** [chakra-ui/panda#2462] [Q&A]
Working on nx 18.2 and seems like both cva and sva are not working ... DO you have any idea ?

**Comment by astahmer** [comment]
havent heard any other similar issue yet, what about the previous version (0.36.1) ?
either way, a minimal reproduction would help both you & me

**Reply by Fazla-GroM** [reply]
Hey @astahmer 👋 
Im having the same issues. 

Here's the [repro](https://stackblitz.com/edit/vitejs-vite-wx7ztp?file=src%2FApp.tsx)

**Reply by astahmer** [reply]
seems like there's an issue with the detection of valid panda imports when generating the `outdir` in a nested folder, in the meantime you can fix it by using `importMap: "/styled-system"`

**Reply by Fazla-GroM** [reply]
Yep, can confirm that adding `importMap` fixed the issue for me 👍
